_authors/ronmiller.markdown

```diff
@@ -0,0 +1,9 @@
+---
+name: Ron Miller
+short_name: ronmiller
+photo: '/assets/media/authors/ronmiller.png'
```

Comment:
Hey, this file says `.png` but the file included is `.jpg`. Would you mind updating?

Reply:
All set. Thank you